### PR TITLE
Collapsible groups

### PR DESCRIPTION
**Pull request by jvincilione**
…-list.

**Merge request checklist**

- [X] I read the [guidelines for contributing](https://github.com/mistic100/jQuery-QueryBuilder/blob/master/.github/CONTRIBUTING.md)
- [X] I created my branch from `dev` and I am issuing the PR to `dev`
- [X] Unit tests are OK
- [X] If it's a new feature, I added the necessary unit tests
- [x] If it's a new language, I filled the `__locale` and `__author` fields

Looks like this solves #347

### CODE REVIEW

**Comment by jvincilione** [comment]
@mistic100 It looks like travis failed due to an error installing sass

```
Error installing sass:
	sass requires Ruby version >= 2.0.0.
```

**Comment by mistic100** [comment]
I will change grunt-contrib-sass to grunt-sass during the week. In order to use node-sass and remove the dependency to Ruby

**Comment by mistic100** [comment]
@jvincilione I switched everythin to node-sass https://github.com/mistic100/jQuery-QueryBuilder/commit/81175dd6fe7e1ada7e69cabd44268932629c5fd7
please rebase your branch

**Comment by jvincilione** [comment]
Nice, looks like everything passed!

**Comment by mistic100** [comment]
Why did you merged your own branch into itself ? When doing a rebase you have to do a force push, instead you get this horrible history.

You can also use interractive rebase to squash your commits.

**Comment by jvincilione** [comment]
Sorry, I'll try and fix it. The merge happened because it wouldn't let me push without pulling down first. I'm not all that familiar with Rebase.

**Comment by jvincilione** [comment]
Fixed it. Sorry for the trouble.

**Comment by jvincilione** [comment]
Added naming support #535

File: src/public.js [hunk 1]
```diff
@@ -237,6 +237,14 @@ QueryBuilder.prototype.getRules = function(options) {
             groupData.data = $.extendext(true, 'replace', {}, group.data);
         }
 
+        if (typeof group.collapsed !== 'undefined') {
```
Comment (mistic100) [inline src/public.js]: this code must no be in the core. use the `groupToJson` event from the plugin

**Comment by jvincilione** [comment]
Made the changes. Let me know if there are any other issues.

**Comment by jvincilione** [comment]
@mistic100 Found a bug with loading. Let me know if there is anything else you want me to do here...

**Comment by mistic100** [comment]
Yes i have other reviews. Sorry for taking so long, my free time is sparse.

**Comment by mistic100** [comment]
@jvincilione Sorry the review was not published, I didn't see it because the behavior changed some months ago

**Pull request closed without merging**